proto/cosmos/auth/v1beta1/tx.proto

```diff
@@ -41,3 +45,18 @@ message MsgUpdateParams {
 //
 // Since: cosmos-sdk 0.47
 message MsgUpdateParamsResponse {}
+
+// MsgAsyncMultiMsgExec defines the Msg/AsyncMultiMsgExec request type.
+message MsgAsyncMultiMsgExec {
+  option (cosmos.msg.v1.signer) = "signer";
+
+  string   signer                   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  repeated google.protobuf.Any msgs = 2 [(cosmos_proto.accepts_interface) = "cosmos.base.v1beta1.Msg"];
+}
+
+// MsgAsyncMultiMsgExecResponse defines the response structure for executing a
+// MsgAsyncMultiMsgExec response.
+message MsgAsyncMultiMsgExecResponse {
```

Comment:
let's create a message.

The message should be: 
```protobuf
message MsgAsyncExecResponse {
   oneof response {
    string error = 1;
    google.protobuf.Any resp = 2;
  }
}
```

We could also avoid the oneof, and contionally assume that if error is empty then we can expect response to be populated with the message response.